build.sbt

```diff
@@ -27,7 +27,7 @@ concurrentRestrictions in Global := Seq(
   Tags.limit(Tags.ForkedTestGroup, testConcurrency)
 )
 
-def groupByHash(tests: Seq[TestDefinition]) = {
+def groupByHash(tests: Seq[TestDefinition]): Seq[Tests.Group] = {
```

Comment:
Why this change?

Reply:
Not a big deal but without the explicit type intelliJ always shows a formatting error. This is just to get rid of that eyesore!